Scripts/capture-build-details.sh

```diff
@@ -5,27 +5,78 @@
 #
 #  Copyright © 2019 LoopKit Authors. All rights reserved.
 
-echo "Gathering build details in ${SRCROOT}"
-cd "${SRCROOT}"
+SCRIPT="$(basename "${0}")"
+SCRIPT_DIRECTORY="$(dirname "${0}")"
 
+error() {
+  echo "ERROR: ${*}" >&2
+  echo "Usage: ${SCRIPT} [-r|--git-source-root git-source-root] [-p|--provisioning-profile-path provisioning-profile-path] [-i|--info-plist-path info-plist-path]" >&2
+  echo "Parameters:" >&2
+  echo "  -r|--git-source-root <git-source-root> root location of git repository to gather information from; optional, defaults to \$WORKSPACE_ROOT if present, otherwise defaults to \$SRCROOT" >&2
+  echo "  -p|--provisioning-profile-path         path to the .mobileprovision provisioning profile file to check for expiration; optional, defaults to \${HOME}/Library/MobileDevice/Provisioning Profiles/\${EXPANDED_PROVISIONING_PROFILE}.mobileprovision" >&2
```

Comment:
[nit] Missing parameter `<provisioning-profile-path>` here and `<info-plist-path>` on the line below.